test/integration/scheduler/scheduler_test.go

```diff
@@ -561,7 +561,12 @@ func TestNodeEvents(t *testing.T) {
 	// 4. Remove the taint from node2; pod2 should now schedule on node2
 
 	testCtx := testutils.InitTestSchedulerWithNS(t, "node-events")
-	defer testCtx.ClientSet.CoreV1().Nodes().DeleteCollection(context.TODO(), metav1.DeleteOptions{}, metav1.ListOptions{})
+	defer func() {
+		err := testCtx.ClientSet.CoreV1().Nodes().DeleteCollection(testCtx.Ctx, metav1.DeleteOptions{}, metav1.ListOptions{})
+		if err != nil {
+			t.Fatalf("Failed to delete all node: %v", err)
+		}
+	}()
```

Comment:
I guess we don't have to clean up nodes actually?
https://github.com/kubernetes/kubernetes/blob/712724634434cbf493ea48de6e711ab96e710cf3/test/integration/util/util.go#L548-L550

Reply:
After reading the source code, it is indeed not needed. I will remove it